devoxx-2018-pl/exhibition/src/main/resources/steps.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
file steps.json is no longer used. The configuration of step order and configuration of the steps is part of `tweetwallConfig.json` via key `stepEngine.steps`. Please remove file and if desired update `tweetwallConfig.json`.